mender-client-acceptance-testing/Dockerfile

```diff
@@ -8,9 +8,9 @@ RUN mkdir -p /mnt/sstate-cache
 # install pyyaml via pip to workardound a later conflict if installed via OS package (awscli depends on python3-yaml)
 RUN apt-get update -qq && apt-get install -yyq python3-pip && pip3 install --upgrade pip && pip3 install pyyaml
 
-# basic tools
-COPY deb-requirements.txt .
-RUN apt-get install -yyq $(cat deb-requirements.txt) && apt-get install -yq linux-modules-`uname -r`
+# Get OS requirements from master
+RUN wget https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/requirements_deb.txt && \
```

Comment:
you most likely already know that you could use an `ADD` to target the remote file and get it transferred - but considering all the caveats docker states around that functionality I think your implementation is the preferred one :+1: